task34.py

```diff
@@ -0,0 +1,18 @@
+""" Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. Поскольку разобраться в его кричалках не настолько просто, насколько легко он их придумывает, Вам стоит написать программу. Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами. Фразы отделяются друг от друга пробелами. Стихотворение  Винни-Пух вбивает в программу с клавиатуры. В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке
+
+*Пример:*
+
+**Ввод:** поро-рл-рам рам-пам-папам па-ра-па-да    
+    **Вывод:** Парам пам-пам  """
+a = input("Введите текст: ").split()
+print(a)
+def same_by(func, a):
+    for i in range(1, len(a)):
+        if func(a[i]) == func(a[i-1]):
+            return True
+    return False
+
+if same_by(lambda x: 1 if 'а' or 'е' or 'у' or 'ы' or 'о' or 'э' or 'я' or 'и' or 'ю' in x else 0, a):
```

Comment:
Ольга, так оператор `in` в связке с `or` не работает. Вам нужно было сделать так:
```suggestion
if same_by(lambda x: 1 if 'а' in x or 'е' in x or 'у' in x or 'ы' in x or 'о' in x or 'э' in x or 'я' in x or 'и' in x or 'ю' in x else 0, a):
```
Но, согласитесь, что возникает такое ощущение, что можно сделать по другому. И действительно можно.
```suggestion
if same_by(lambda x: 1 if x in 'аяуюоеёэиы' else 0, a):
```